cli/src/commands/subgraph/commands/fetch.ts

```diff
@@ -0,0 +1,47 @@
+import { writeFile } from 'node:fs/promises';
+import { EnumStatusCode } from '@wundergraph/cosmo-connect/dist/common/common_pb';
+import { Command } from 'commander';
+import { join } from 'pathe';
+import pc from 'picocolors';
+import { baseHeaders } from '../../../core/config.js';
+import { BaseCommandOptions } from '../../../core/types/types.js';
+import program from '../../index.js';
+
+export default (opts: BaseCommandOptions) => {
+  const command = new Command('fetch');
+  command.description('Fetches the latest valid SDL of a subgraph. The output can be piped to a file.');
+  command.argument('<name>', 'The name of the subgraph to fetch.');
+  command.option('-n, --namespace [string]', 'The namespace of the subgraph.');
+  command.option('-o, --out [string]', 'Destination file for the SDL.');
+  command.action(async (name, options) => {
+    const resp = await opts.client.platform.getLatestValidSubgraphSDLByName(
+      {
+        name,
```

Comment:
The current implementation expects the name of the federated graph. Please see https://github.com/wundergraph/cosmo/pull/564#issuecomment-1964537496